### ISSUE TITLE
ModuleNotFoundError: No module named 'multipart' when using Gradio Lite via CDN

### ISSUE THREAD

**Issue by k-kondo-s** [gradio-app/gradio#9839]
### Describe the bug

Related to #9774

When trying to use Gradio Lite via CDN as documented, the application fails to load due to a missing 'multipart' dependency. This appears to be a dependency management issue with Pyodide/Gradio Lite setup.

Additionally, the official Gradio Lite documentation page itself appears to be broken and not rendering correctly.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```html
<!DOCTYPE html>
<html>
    <head>
        <script type="module" crossorigin src="https://cdn.jsdelivr.net/npm/@gradio/lite/dist/lite.js"></script>
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@gradio/lite/dist/lite.css" />
    </head>
    <body>
        <gradio-lite>
            <!-- Gradio application code here -->
        </gradio-lite>
    </body>
</html>
```

### Screenshot

<img width="1917" alt="image" src="https://github.com/user-attachments/assets/a86d818e-ddba-4c69-8802-5e51f96bdfce">

<img width="1908" alt="image" src="https://github.com/user-attachments/assets/ba54dd76-cbde-4e94-b501-40b9569baada">

### Logs

```shell
Traceback (most recent call last): File "/lib/python312.zip/_pyodide/_base.py", line 574, in eval_code_async await CodeRunner( File "/lib/python312.zip/_pyodide/_base.py", line 394, in run_async coroutine = eval(self.code, globals, locals) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "<exec>", line 1, in <module> File "/lib/python3.12/site-packages/gradio/__init__.py", line 3, in <module> import gradio._simple_templates File "/lib/python3.12/site-packages/gradio/_simple_templates/__init__.py", line 1, in <module> from .simpledropdown import SimpleDropdown File "/lib/python3.12/site-packages/gradio/_simple_templates/simpledropdown.py", line 7, in <module> from gradio.components.base import Component, FormComponent File "/lib/python3.12/site-packages/gradio/components/__init__.py", line 1, in <module> from gradio.components.annotated_image import AnnotatedImage File "/lib/python3.12/site-packages/gradio/components/annotated_image.py", line 14, in <module> from gradio import processing_utils, utils File "/lib/python3.12/site-packages/gradio/processing_utils.py", line 34, in <module> from gradio.route_utils import API_PREFIX File "/lib/python3.12/site-packages/gradio/route_utils.py", line 34, in <module> import multipart ModuleNotFoundError: No module named 'multipart'

PythonError: Traceback (most recent call last):
  File "/lib/python312.zip/_pyodide/_base.py", line 574, in eval_code_async
    await CodeRunner(
  File "/lib/python312.zip/_pyodide/_base.py", line 394, in run_async
    coroutine = eval(self.code, globals, locals)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<exec>", line 1, in <module>
  File "/lib/python3.12/site-packages/gradio/__init__.py", line 3, in <module>
    import gradio._simple_templates
  File "/lib/python3.12/site-packages/gradio/_simple_templates/__init__.py", line 1, in <module>
    from .simpledropdown import SimpleDropdown
  File "/lib/python3.12/site-packages/gradio/_simple_templates/simpledropdown.py", line 7, in <module>
    from gradio.components.base import Component, FormComponent
  File "/lib/python3.12/site-packages/gradio/components/__init__.py", line 1, in <module>
    from gradio.components.annotated_image import AnnotatedImage
  File "/lib/python3.12/site-packages/gradio/components/annotated_image.py", line 14, in <module>
    from gradio import processing_utils, utils
  File "/lib/python3.12/site-packages/gradio/processing_utils.py", line 34, in <module>
    from gradio.route_utils import API_PREFIX
  File "/lib/python3.12/site-packages/gradio/route_utils.py", line 34, in <module>
    import multipart
ModuleNotFoundError: No module named 'multipart'

    at new_error (https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.js:10:9965)
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[302]:0x16dbfd
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[489]:0x177431
    at _PyEM_TrampolineCall_JS (https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.js:10:125894)
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[1134]:0x1c2eaf
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[3602]:0x2c7c0f
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[2167]:0x20a884
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[1142]:0x1c359c
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[1145]:0x1c38ab
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[1146]:0x1c3929
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[3408]:0x29e95d
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[3409]:0x2a4f54
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[1148]:0x1c3a69
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[1143]:0x1c36d2
    at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.wasm:wasm-function[476]:0x176a95
    at callPyObjectKwargs (https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.js:10:64068)
    at Module.callPyObjectMaybePromising (https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.js:10:65316)
    at wrapper (https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.js:10:27006)
    at Cn.e.port1.onmessage (https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.js:10:102154)
```
```


### System Info

```shell
OS: macOS 14.6.1 (23G93)
Browser: Chrome 130.0.6723.60
Gradio Lite: Latest version from CDN
```

### Severity

Blocking usage of gradio

**Comment by freddyaboulton** [comment]
Should be fixed by #9834 in the next release

**Comment by freddyaboulton** [comment]
Should be fixed by #9834 in the next release

**Comment by freddyaboulton** [comment]
Should be fixed in version 5.4.0 of gradio/lite!
